admin/query/src/main/java/org/codice/ditto/replication/admin/query/ReplicationUtils.java

```diff
@@ -132,37 +138,43 @@ public ReplicationField updateReplication(
                             + name
                             + ". No configuration found."));
 
-    ReplicatorConfigImpl config = new ReplicatorConfigImpl(existingConfig);
-    config.setName(name);
-    config.setSource(sourceId);
-    config.setDestination(destinationId);
-    config.setCql(filter);
-    config.setDirection(biDirectional ? Direction.BOTH : Direction.PUSH);
```

Comment:
❗️ Right now ReplicatorImpl still only uses the direction field. We either need to set it here or update the ReplicatorImpl to also look at the bi-directional field.

Reply:
ReplicatorImpl still works because were not using the bidirectional field just yet. I'm aiming to make that change in #36. Here's what the bidirectional methods currently look like 

` @Override
    this.cql = cql;
  public boolean isBiDirectional() {
    return direction == Direction.BOTH;
  }

  @Override
  public void setBiDirectional(boolean biDirectional) {
    direction = biDirectional ? Direction.BOTH : Direction.PUSH;
  }`